### DISCUSSION TITLE
Difficulties in hosting peerjs on a cPanel

### DISCUSSION THREAD

**Discussion by amjidkhanmohmand** [peers/peerjs#1215] [Q&A]
Hello team PeerJs
I am using Namecheap hosting to host my app. I have set up the nodejs app so far and installed the peerjs dependency.  
### Now I am facing some issues:

- when I access the app from one laptop in two different tabs in a browser it works fine
- but when I try to use the app using two different laptops it does not work at all.

### The required support:

- Kindly provide some helpful resources so that I can setup my app, I have gone through the documentation but unfortunately I was unable to find the resource that I am looking for, kindly help me out here.

Thanks and Regards:
Amjid Khan Mohmand

**Comment by jonasgloning** [comment]
Hi @amjidkhanmohmand.
Let's first ensure this is a problem with the server and not the client.
Does it work with the public server (Without specifying `host` or `port`)?

**Reply by amjidkhanmohmand** [reply]
Hi @jonasgloning 
Thanks for the reply. I want to use my own server as a host. Is it possible to use my own server instead of the public server? I am attaching the code that I am using on my local server (my own PC---I mean localhost). What I am trying to do is to replace the host and port to my online server so that I can integrate it in my mobile app. This is the link to the sample code that I am using https://drive.google.com/drive/folders/1MdOYXhs-AF5Z3sKMmx-7ovkgfzhyUqId?usp=share_link

**Reply by jonasgloning** [reply]
Sure, as long as it's reachable by all clients.
You hardcoded the server ip in your client code. Are you sure it's reachable by both laptops? Also no firewall that blocks access to the port?
```ts
peer = new Peer(userId, {
        host: '192.168.18.4',
        ...
    })
```

**Reply by amjidkhanmohmand** [reply]
This is only accessable by the devices within the same network. I want to replace it with my online server. How can I do it? Please provide an example directory if possible. If not, kindly provide some documentation.
To be more specific I want to replace host with my domain name e.g. https://www.example.com

Note: 
I am using GoDaddy's dedicated server to setup my own streaming link.
